.github/test-spec.yml

```diff
@@ -85,21 +85,17 @@
     - "!openthread/doc/**/*"
 
 "CI-nfc-test":
-  - "**/*"
-  - "!softdevice_controller/include/**/*"
-  - "!softdevice_controller/lib/**/*"
-  - "!mpsl/include/**/*"
-  - "!mpsl/lib/**/*"
+  - "nfc/**/*"
 
 "CI-matter-test":
   - "mpsl/include/**/*"
   - "mpsl/lib/**/*"
 
 "CI-find-my-test":
-  - "**/*"
+  - "!nrf_modem/**/*"
```

Comment:
@kapi-no could you advise a better pattern?

Reply:
sure